CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `ModeSolverData.pol_fraction` and `ModeSolverData.pol_fraction_waveguide` properties to compute polarization fraction of modes using two different definitions.
 - `ModeSolverData.to_dataframe()` and `ModeSolverData.modes_info` for a convenient summary of various modal properties of the computed modes.
 - Loss upper bound estimation in `PoleResidue` material model.
+- `tidy3d/converter.py` for conversion from .lsf project files into Tidy3D files through command line, e.g. `python -m tidy3d convert example.lsf tidy3d_script.py` or `tidy3d convert example.lsf tidy3d_script.py`
```

Comment:
let's change this to just reflect the `tidy3d convert`. I dont think we should support both. And also instead of "Tidy3D files" maybe "equivalent python scripts implementing the project in Tidy3D".